index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta property="og:image" content="./screen.png" />
+    <meta
+      property="og:description"
+      content="This is a Webpack Project about Todo-list which you can add your to-dos, edit them, add mark as completed and store the data in localStorage"
+    />
+    <title>Todo list</title>
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+    <script defer src="main.bundle.js"></script>
+  </head>
+
+  <body>
+    <div class="container">
+      <h4 class="title">Today's Todo</h4>
+      <form>
+        <input
+          class="input-text"
+          type="text"
+          placeholder="Add to your List..."
+          autofocus
+        />
+        <button class="add-btn" type="button" >
+          <i class="fas fa-location-arrow"></i>
+        </button>
+      </form>
+      <div class="todos"></div>
+      <div class="clear">
+        <a class="clear-btn">Clear All</a>
+      </div>
+    </div>
```

Comment:
Please try to follow the HTML semantics to apply best practices in HTML. For more information please [checkout this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main)